requirements.txt

```diff
@@ -0,0 +1,4 @@
+flask==0.12.2
```

Comment:
Probably one for another card but as it's a new project, let's make sure we're using [Pipenv](https://docs.pipenv.org/index.html).